Remove the following

 2011-04-20  Richard Sandiford  <richard.sandiford@linaro.org>
 
	gcc/testsuite/
	From  Richard Earnshaw  <rearnsha@arm.com>

	PR target/46329
	* gcc.target/arm/pr46329.c: New test.

=== removed file 'gcc/testsuite/gcc.target/arm/pr46329.c'
--- old/gcc/testsuite/gcc.target/arm/pr46329.c	2011-05-03 12:49:58 +0000
+++ new/gcc/testsuite/gcc.target/arm/pr46329.c	1970-01-01 00:00:00 +0000
@@ -1,11 +0,0 @@
-/* { dg-do compile } */
-/* { dg-require-effective-target arm_neon_ok } */
-/* { dg-options "-O2" } */
-/* { dg-add-options arm_neon } */
-
-int __attribute__ ((vector_size (32))) x;
-void
-foo (void)
-{
-  x <<= x;
-}

